### PR TITLE
[Google Workspace] DNS record checklist

### PR DESCRIPTION
**Pull request by rluodev**
## Summary of the problem

It's kinda difficult for users to figure out what they're missing - this leads to additional user friction and takes up HCB ops capacity to answer questions.

## Describe your changes

When a verification error occurs, a new `Found` column will appear at the start of the table that has `✅` or `❌` based on if HCB can find the record. This will help users figure out wtf is going on with their dns

Screenshot 2024-06-12 at 1 39 46 PM

Screenshot 2024-06-12 at 1 39 11 PM

### CODE REVIEW

**Review by sarthaktexas — DISMISSED** [review]

File: app/controllers/events_controller.rb
Comment (sampoder) [inline app/controllers/events_controller.rb]: Instead of an array, could we use a hash?

**Comment by rluodev** [comment]
# PLEASE DO NOT MERGE

gary wants this to look more like a checklist

File: app/controllers/events_controller.rb
Comment (rluodev) [reply app/controllers/events_controller.rb]: i kinda just did it like this for simplicity cause I can just take the index of the array

Comment (sampoder) [reply app/controllers/events_controller.rb]: To someone who didn't write the code, it's really confusing when it's presented like this

Comment (rluodev) [reply app/controllers/events_controller.rb]: hmm true, I'll change this to a hash

Comment (rluodev) [reply app/controllers/events_controller.rb]: oh wait no nvm i just remembered i also did it this way to make it simpler for the mx record checking, cause this way I can just take the mx record index and add two to it to change the result array index to true if found

Comment (sampoder) [reply app/controllers/events_controller.rb]: This either needs to be commented and explained clearly or implemented in a cleaner way. I'd strongly prefer for it to be a hash.

File: app/controllers/events_controller.rb
Comment (garyhtou) [inline app/controllers/events_controller.rb]:
```suggestion
    # this is janky and should be fixed at some point!
    @result = [false, false, false, false, false, false, false]
```

File: app/views/events/g_suite_overview.html.erb
Comment (garyhtou) [inline app/views/events/g_suite_overview.html.erb]:
```suggestion
                    <th></th>
```

**Review by garyhtou — COMMENTED** [review]
🪄 good to merge after adding a stikethrough on completed rows

**Comment by rluodev** [comment]
@garyhtou done!

File: app/assets/stylesheets/_utilities.scss
Comment (garyhtou) [inline app/assets/stylesheets/_utilities.scss]:
```suggestion
}

```

**Pull request merged**
